src/apps/external_executor/protobuf/executor_registration.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package ccf;
+package externalexecutor.protobuf;
```

Comment:
I think `.protobuf` here is a little verbose and obvious isn't it? Especially as it ends up in the route for all endpoints in this app?